src/components/hooks/useAccordion/useAccordion.ts

```diff
@@ -2,6 +2,40 @@ import { useRef, useState, Dispatch, SetStateAction } from 'react';
 import { isArrowDown, isArrowUp } from '../../../helpers/keyboard-keys';
 import { mod } from '../../../helpers/utils';
 
+interface IAccordionHeaderProps {
+  'aria-controls': string;
+  'aria-disabled': boolean;
+  'aria-expanded': boolean;
+  id: string;
+  key: string;
+  onClick: () => void;
+  onFocus: () => void;
+  onKeyDown: (e: React.KeyboardEvent<HTMLButtonElement>) => void;
```

Comment:
interesting... React is not imported but still works 🤔

Reply:
no jsx :)

Reply:
But still `React.KeyboardEvent` is not imported but I guess because React refers to the namespace in the type definitions instead of the React object itself? 🤔

Reply:
oh yeah, you're right... hmmm 🤔

Reply:
imported it, just in case